packs/st2ask/actions/workflows/two_reponses_roles.yaml

```diff
@@ -0,0 +1,54 @@
+---
+version: '2.0'
+
+st2ask.two_responses_roles:
+    description: An st2.ask workflow that demonstrates two responses from any user in the specified roles
+    type: direct
+
+    task-defaults:
+        on-error:
+            - rejected
+
+    tasks:
+
+        init:
+            action: core.noop
+            on-complete:
+                - manager_approval
+                - developer_approval
+
+        # This st2.ask example illustrates how multiple responders can be used to satisfy a workflow
+        # In this case, we create two separate st2.ask tasks, one for each approver.
+        manager_approval:
+            action: st2.ask
+            input:
+                tag: managers
+
+                # The "roles" param is one way of restricting who is able to respond to this instance of st2.ask
+                # In this case, only those in a role labeled "managers" will be allowed to respond to this action
+                #
+                # NOTE this requires RBAC support, which is only available in the Enterprise version of StackStorm.
+                # See the "users" param for a similar way that doesn't require RBAC
+                roles:
+                    - managers
+
+        developer_approval:
+            action: st2.ask
+            input:
```

Comment:
Hrm, so we're abandoning the idea of doing a default schema for simple approvals? Figured since it was a common use case it made sense to make it as easy as possible to use. And in this case, how would an `st2.ask` action be rejected? Or are we just moving totally away from the approve/reject verbiage and going for a purely generic ask/respond? Obviously  it would be the latter under the covers but I was under the impression we wanted to make the specific approve/reject use case a bit easier since it was so common.